descriptors/ModuleDescriptor-template.json

```diff
@@ -17,6 +17,39 @@
           ],
           "modulePermissions": [
           ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/data-export/fileDefinitions/{fileDefinitionId}",
+          "permissionsRequired": [
+            "data-export.filedefinitions.get"
+          ],
+          "modulePermissions": [
+          ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/data-export/fileDefinitions/{fileDefinitionId}",
+          "permissionsRequired": [
+            "data-export.filedefinitions.uploadfile"
```

Comment:
please name the permissions in line with othe rmodules. EAch entry post is usually named item.post